packages/components/future/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "main": "../dist/__future__/index.js",
-  "module": "../dist/esm/__future__/index.js",
-  "types": "../dist/__future__/index.d.ts"
+  "main": "../dist/cjs/future.js",
+  "module": "../dist/esm/future.js",
+  "types": "../dist/esm/dts/__future__/index.d.ts"
```

Comment:
This is in a weird place though lol

Reply:
Yeah I figured we could take a crack at updating the output path or creating a future.d.ts when we loop back to the tooling health day :)